react/components/Iframe.js

```diff
@@ -87,27 +87,27 @@ class Iframe extends Component {
 
   updateBrowserHistory = ({ pathname: iframePathname, search: iframeSearch, hash: iframeHash }) => {
     const { isDeloreanAdmin } = this.props
-    const { navigate } = this.context
     const { pathname, search = '', hash = '' } = window.location
     const patchedIframeSearch = iframeSearch.replace(/(\?|\&)env\=beta/, '')
 
-    if (iframePathname.replace(isDeloreanAdmin ? '/iframe' : '/app', '') !== pathname
+    let formatedIframePathname = iframePathname.replace(isDeloreanAdmin ? '/iframe' : '/app', '')
+
+    const formatedPathname = !pathname.endsWith('/') ? `${pathname}/` : pathname
```

Comment:
nit: could be a function `withTrailingSlash()`

Reply:
Also, wouldn't it be better to _remove_ the trailing slash by default? Apparently this thingy here which is used by most admins only adds trailing slash if there is a hash: https://io.vtex.com.br/oms-ui/2.31.1/lib/vtex-verify-url-slash.js